src/adu_types/inc/aduc/adu_types.h

```diff
@@ -7,9 +7,11 @@
 #ifndef ADUC_ADU_TYPES_H
```

Comment:
>ADUC_ADU_TYPES_H [](start = 8, length = 16)

this file seems to be around launch argument types, and not "common data types" used throughout ADU (line 3)